po/documentation.de.po

```diff
@@ -8,8 +8,13 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: git documentation\n"
+<<<<<<< HEAD
 "POT-Creation-Date: 2019-01-26 17:40+0100\n"
 "PO-Revision-Date: 2019-01-16 13:41+0100\n"
+=======
```

Comment:
How did these get past the mergetool? Idk. Well, I'll have to push an update anyways.

Reply:
Ok, I've messed this up, again. Great.

Reply:
This is due to the wildcards in the Makefile:

```Makefile
LANGUAGE_PO = $(wildcard po/*.po)
```

That catches your special po file.